### DISCUSSION TITLE
[VSCode] Multiple cursor insert code suggestion

### DISCUSSION THREAD

**Discussion by samuel-tonini** [community/community#13980] [Copilot Conversations]
When I use mulple cursor and Copilot give me an pieace of code, the code was only inserted on the initial location.

https://user-images.githubusercontent.com/11463377/161070079-9988acfc-274d-4b14-ab4e-9eabca214b19.mov

**Comment by beaver700nh** [comment]
I agree with you, it should suggest completion for all current active cursors, not just the first one. The fact that it doesn't already to this is a bit surprising, as the builtin autocomplete as well as other extensions like C/C++ Intellisense use all the cursors as well.
